[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/selenium/webdriver/common/macos/selenium-manager
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /Users/sumit/PycharmProjects/PartnerPortalUI/.venv/lib/python3.9/site-packages/selenium/webdriver/common/macos/selenium-manager --browser chrome --debug --language-binding python --output json
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chromedriver not found in PATH
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --version
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.114 "
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.114
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.94
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chromedriver 136.0.7103.94 already in the cache
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /Users/sumit/.cache/selenium/chromedriver/mac-arm64/136.0.7103.94/chromedriver
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
[35mDEBUG   [0m selenium.webdriver.common.service:service.py:231 Started executable: `/Users/sumit/.cache/selenium/chromedriver/mac-arm64/136.0.7103.94/chromedriver` in a child process with pid: 89264 using 0 to output -3
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 POST http://localhost:62114/session {'capabilities': {'firstMatch': [{}], 'alwaysMatch': {'browserName': 'chrome', 'pageLoadStrategy': <PageLoadStrategy.normal: 'normal'>, 'browserVersion': None, 'goog:chromeOptions': {'extensions': [], 'binary': '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome', 'args': ['--start-maximized', '--disable-notifications']}}}}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:62114
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 http://localhost:62114 "POST /session HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:436 Remote response: status=200 | data={"value":{"capabilities":{"acceptInsecureCerts":false,"browserName":"chrome","browserVersion":"136.0.7103.114","chrome":{"chromedriverVersion":"136.0.7103.94 (fa0be0b33debeb378a8e6ad9c599be34e2dc3b37-refs/branch-heads/7103@{#1842})","userDataDir":"/var/folders/tf/qc7qzl6x29gdc6yq7s8dn5f80000gp/T/.org.chromium.Chromium.10WDVz"},"fedcm:accounts":true,"goog:chromeOptions":{"debuggerAddress":"localhost:62119"},"networkConnectionEnabled":false,"pageLoadStrategy":"normal","platformName":"mac","proxy":{},"setWindowRect":true,"strictFileInteractability":false,"timeouts":{"implicit":0,"pageLoad":300000,"script":30000},"unhandledPromptBehavior":"dismiss and notify","webauthn:extension:credBlob":true,"webauthn:extension:largeBlob":true,"webauthn:extension:minPinLength":true,"webauthn:extension:prf":true,"webauthn:virtualAuthenticators":true},"sessionId":"2614be03a721d27312a893229b5cc12e"}} | headers=HTTPHeaderDict({'Content-Length': '891', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:465 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 POST http://localhost:62114/session/2614be03a721d27312a893229b5cc12e/timeouts {'implicit': 10000}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 http://localhost:62114 "POST /session/2614be03a721d27312a893229b5cc12e/timeouts HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:436 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:465 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 POST http://localhost:62114/session/2614be03a721d27312a893229b5cc12e/url {'url': 'https://myyatra.finkraft.ai/auth/signin'}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 http://localhost:62114 "POST /session/2614be03a721d27312a893229b5cc12e/url HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:436 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:465 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 POST http://localhost:62114/session/2614be03a721d27312a893229b5cc12e/window/maximize {}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 http://localhost:62114 "POST /session/2614be03a721d27312a893229b5cc12e/window/maximize HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:436 Remote response: status=200 | data={"value":{"height":884,"width":1512,"x":0,"y":38}} | headers=HTTPHeaderDict({'Content-Length': '50', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:465 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 POST http://localhost:62114/session/2614be03a721d27312a893229b5cc12e/element {'using': 'xpath', 'value': "//input[@placeholder='Email']"}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 http://localhost:62114 "POST /session/2614be03a721d27312a893229b5cc12e/element HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:436 Remote response: status=200 | data={"value":{"element-6066-11e4-a52e-4f735466cecf":"f.043E44D5557557E28A3EE19839A55A82.d.408902353904F6A0FF41F970595EA094.e.11"}} | headers=HTTPHeaderDict({'Content-Length': '126', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:465 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 POST http://localhost:62114/session/2614be03a721d27312a893229b5cc12e/execute/sync {'script': '/* isDisplayed */return (function(){return (function(){var g=this||self;\nfunction aa(a){var b=typeof...', 'args': [{'element-6066-11e4-a52e-4f735466cecf': 'f.043E44D5557557E28A3EE19839A55A82.d.408902353904F6A0FF41F970595EA094.e.11'}]}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 http://localhost:62114 "POST /session/2614be03a721d27312a893229b5cc12e/execute/sync HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:436 Remote response: status=200 | data={"value":true} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:465 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 POST http://localhost:62114/session/2614be03a721d27312a893229b5cc12e/element/f.043E44D5557557E28A3EE19839A55A82.d.408902353904F6A0FF41F970595EA094.e.11/value {'text': 'wronguser', 'value': ['w', 'r', 'o', 'n', 'g', 'u', 's', 'e', 'r']}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 http://localhost:62114 "POST /session/2614be03a721d27312a893229b5cc12e/element/f.043E44D5557557E28A3EE19839A55A82.d.408902353904F6A0FF41F970595EA094.e.11/value HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:436 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:465 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 POST http://localhost:62114/session/2614be03a721d27312a893229b5cc12e/element {'using': 'xpath', 'value': '//button[@type="submit"]'}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 http://localhost:62114 "POST /session/2614be03a721d27312a893229b5cc12e/element HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:436 Remote response: status=200 | data={"value":{"element-6066-11e4-a52e-4f735466cecf":"f.043E44D5557557E28A3EE19839A55A82.d.408902353904F6A0FF41F970595EA094.e.15"}} | headers=HTTPHeaderDict({'Content-Length': '126', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:465 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 POST http://localhost:62114/session/2614be03a721d27312a893229b5cc12e/execute/sync {'script': '/* isDisplayed */return (function(){return (function(){var g=this||self;\nfunction aa(a){var b=typeof...', 'args': [{'element-6066-11e4-a52e-4f735466cecf': 'f.043E44D5557557E28A3EE19839A55A82.d.408902353904F6A0FF41F970595EA094.e.15'}]}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 http://localhost:62114 "POST /session/2614be03a721d27312a893229b5cc12e/execute/sync HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:436 Remote response: status=200 | data={"value":true} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:465 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 GET http://localhost:62114/session/2614be03a721d27312a893229b5cc12e/element/f.043E44D5557557E28A3EE19839A55A82.d.408902353904F6A0FF41F970595EA094.e.15/enabled {}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 http://localhost:62114 "GET /session/2614be03a721d27312a893229b5cc12e/element/f.043E44D5557557E28A3EE19839A55A82.d.408902353904F6A0FF41F970595EA094.e.15/enabled HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:436 Remote response: status=200 | data={"value":true} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:465 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 POST http://localhost:62114/session/2614be03a721d27312a893229b5cc12e/element/f.043E44D5557557E28A3EE19839A55A82.d.408902353904F6A0FF41F970595EA094.e.15/click {}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 http://localhost:62114 "POST /session/2614be03a721d27312a893229b5cc12e/element/f.043E44D5557557E28A3EE19839A55A82.d.408902353904F6A0FF41F970595EA094.e.15/click HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:436 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:465 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 POST http://localhost:62114/session/2614be03a721d27312a893229b5cc12e/element {'using': 'xpath', 'value': "//input[@placeholder='Password']"}
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:403 DELETE http://localhost:62114/session/2614be03a721d27312a893229b5cc12e {}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (2): localhost:62114
[35mDEBUG   [0m urllib3.util.retry:retry.py:521 Incremented Retry for (url='/session/2614be03a721d27312a893229b5cc12e'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x118c80a00>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/2614be03a721d27312a893229b5cc12e
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (3): localhost:62114
[35mDEBUG   [0m urllib3.util.retry:retry.py:521 Incremented Retry for (url='/session/2614be03a721d27312a893229b5cc12e'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x118c80670>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/2614be03a721d27312a893229b5cc12e
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (4): localhost:62114
[35mDEBUG   [0m urllib3.util.retry:retry.py:521 Incremented Retry for (url='/session/2614be03a721d27312a893229b5cc12e'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x118c80430>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/2614be03a721d27312a893229b5cc12e
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (5): localhost:62114